tutorials/intro_to_api/projects.md

```diff
@@ -0,0 +1,258 @@
+ userab1cdef2   241     # default project
+```
+
+## Adding a Task to a Project
+
+Any task that is submitted to the Inductiva API must belong to a project.
+When the user is not explicit about the project to which the task is to be added,
+the default one is used. This ensures that no orphaned tasks are created.
+To add a task to a project other than the default one, the user needs to create
+a new project (or reference an existing one) and open it for task submission. However,
+the project can only be opened for task submission if it gets instantiated with
+an explicit `append=True` argument in the constructor or, otherwise, a RunTime
+error will be raised. This mechanism is meant to work as a security measure that
+ensures that the user is explicitly aware that new tasks are being appended to the
+project, which is particularly useful to ensure the consistency of the project's
+content.
+
+```python
+import inductiva
+
+project = inductiva.projects.Project("demo", append=False)
+project.open() # <-- An exception will be raised because the project is appendable
+...
+RuntimeError: Trying to open a project with `append=False`.
+A Project can only be opened when instantiated with the `append=True` option.
+```
+
+Whenever the `run` method of a `Simulator` object is called, the resulting task
+will be created under the project that is currently open, even though the project
+is not explicitly specified as an argument to the `run` method. It will be inferred
+from the context where the `run` method is being called.
+
+The client library offers two mechanisms to manage how tasks are added to a
+project: via a context manager or by manually opening and closing a `project`.
+
+#### Using a Context Manager
+
+The following snippet demonstrates how to add a task to a project using a context
+manager. The manager ensures that the project is opened for task submission
+by calling the `open` and `close` methods when entering and exiting the context
+of the `with` block, respectively. Any task that is created inside the `with` block
+will be appended to the project managed in that context; tasks submitted outside
+will be added to the default one:
+
+```python
+import inductiva
+
+# get example input data
+input_dir = inductiva.utils.download_from_url(
+    "https://storage.googleapis.com/inductiva-api-demo-files/"
+    "xbeach-input-example.zip", unzip=True)
+
+with inductiva.projects.Project("my_xbeach_project", append=True) as project:
+    simulator = inductiva.simulators.XBeach()
+
+    # add a task to the "my_xbeach_project" project
+    task1 = simulator.run(input_dir=input_dir,
+                          sim_config_filename="params.txt")
+
+# task2 will be added to the default project
+task2 = simulator.run(input_dir=input_dir,
+                      sim_config_filename="params.txt")
+
+print(task1.get_info()['project']) # "my_xbeach_project"
+print(task2.get_info()['project']) # "userab1cdef2" (default project)
+```
+
+#### Explicit management
```

Comment:
Maybe we should start with the explicit option.

Reply:
Done in the development branch.